test/test.config

```diff
@@ -23,10 +23,9 @@
        {get_options, [{r, 2}, {pr, 0}]}
      ]}
    ]},
-
    {docs, [
-     {sumo_test_people_riak, sumo_test_riak},
-     {sumo_test_purchase_order, sumo_test_riak}
+     {people, sumo_test_riak, #{module => sumo_test_people_riak}},
+     {purchase, sumo_test_riak, #{module => sumo_test_purchase_order}}
```

Comment:
I think this is totally subjective, what I've seen is that it is named in the same way as its entity, in this case `purchase_order`, but the idea is to allow the users to put what they consider (`purchases`, `purchase`, etc.). We can change it, but again, IMHO it is subjective.